Standard.AI.Data.EntityIntelligence/Services/Foundations/Datas/IDataService.cs

```diff
@@ -0,0 +1,15 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Standard.AI.Data.EntityIntelligence.Models.Datas.Services;
+
+namespace Standard.AI.Data.EntityIntelligence.Services.Foundations.Datas
+{
+    internal interface IDataService
+    {
+        ValueTask<List<TableMetadata>> RetrieveTablesDetailsAsync();
```

Comment:
We discussed this. this comment is just a reminder to change the name of the method here to be `RetrieveTableMetadatasAsync()`